docs/README.md

```diff
@@ -1,3 +1,97 @@
+few of them could help solve complex underlay-network issues, so we decide to develop the Spiderpool.
+
+BTW, there are also some CNI plugins that could work on the underlay mode, such as [kube-ovn](https://github.com/kubeovn/kube-ovn) and [coil](https://github.com/cybozu-go/coil).
+But the Spiderpool provides lots of different features, you could see [Features](#features) for details.
+
+## Features
+
+The Spiderpool provides a large number of different features as follows.
+
+* Based on CRD storage, all operation could be done with kubernetes API-server.
+
+* Support for assigning IP addresses with three options: IPv4-only, IPv6-only, and dual-stack.
+
+* Support for working on the clusters with three options: IPv4-only, IPv6-only, and dual-stack.
+
+* Support for creating multiple ippools.
+  Different namespaces and applications could monopolize or share an ippool.
+
+* An application could specify multiple backup ippool resources, in case that IP addresses in an ippool are out of use. Therefore, you neither need to scale up the IP resources in a fixed ippool, nor need to modify the application yaml to change a ippool.
+
+* Support to bind range of IP address only to an applications. No need to hard code an IP list in deployment yaml, which is not easy to modify. With Spiderpool, you only need to set the selector field of ippool and scale up or down the IP resource of an ippool dynamically.
+
+* Support Statefulset pod who will be always assigned same IP addresses.
+
+* Different pods in a single controller could get IP addresses from
+  different subnets for an application deployed in different subnets or zones.
+
+* Administrator could safely edit ippool resources, the Spiderpool will help validate the modification and prevent from data race.
+
+* Collect resources in real time, especially for solving IP leakage or slow collection, which may make new pod fail to assign IP addresses.
+
+* Support ranges of CNI plugin who supports third-party IPAM plugins. Especially, the Spiderpool could help much for CNI like [spiderflat](https://github.com/spidernet-io/spiderflat),
+  [macvlan CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/macvlan),
+  [vlan CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/vlan),
+  [ipvlan CNI](https://github.com/containernetworking/plugins/tree/main/plugins/main/ipvlan),
+  [sriov CNI](https://github.com/k8snetworkplumbingwg/sriov-cni),
+  [ovs CNI](https://github.com/k8snetworkplumbingwg/ovs-cni).
+
+* Especially support for [Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni) case to assign IP for multiple interfaces.
+
+* Have a good performance for assigning and collecting IP.
+
+* Support to reserve IP who will not be assigned to any pod.
+
+* Included metrics for looking into IP usage and issues.
+
+* By CidrManager, it could automatically scale up and down the IP address of the ippool, to distribute IP resource more reasonable between ippool.
```

Comment:
```suggestion
* By CIDR Manager, it could automatically scale up and down the IP address of the ippool, to distribute IP resource more reasonable between ippool.
```